src/contact/components/DisplayInfo.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+
+class DisplayInfo extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  
+  componentDidMount() {
+    if(this.props.info === null){
+      this.props.actions.displayInfo(false);
+    }
+  }
+
+  onEdit(){
+    this.props.history.push('/edit');
+  }
+
+  render() {
+    //fallback UI
+    if (this.props.hasError || this.props.info === null) {
+      return (
+        <div>
+          <h1>Oops, something went wrong :(</h1>
+          <p>Error in Loading</p>
+        </div>
+      );
+    }
+    if(this.props.info){
```

Comment:
Why you are checking for same value multiple times, you could use ? : operator and render both views

Reply:
Here also, ternary can be used, I have used ternary in some files.